tests/e2e/specs/edit-posts.test.js

```diff
@@ -0,0 +1,143 @@
+import {
+	createNewPost,
+	pressKeyTimes,
+	switchUserToAdmin,
+	switchUserToTest,
+	visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Trash any posts on the edit posts screen.
+ */
+async function trashExistingPosts() {
```

Comment:
It wasn't there when the code was originally written, but this should be able to use `trashAllPosts` from test utils now:
https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-test-utils/src/posts.js#L21

Reply:
Hmm, I'm trying to use it but it errors with `invalid post type`. The default is `post` so I'm not sure what's happening 😕

Reply:
Update: in order to get `trashAllPosts` to work I have to call it within a callback passed to `beforeEach`, but when I try that the admin and password strings get bizarrely mangled:

<img width="549" alt="Screen Shot 2021-03-22 at 1 23 54 pm" src="https://user-images.githubusercontent.com/8096000/111933889-7694c480-8b14-11eb-945f-6f72a5d207f4.png">

Reply:
Interesting, that sounds like what was happening when I tested the visual regression tests:
https://github.com/WordPress/wordpress-develop/pull/209#pullrequestreview-573957665

That gif doesn't show it, but I did see the tests unable to type properly in those input fields.

Reply:
Found the issue: I forgot to `await` `trashAllPosts` 🤦